.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -8,6 +8,9 @@ assignees: ''
 ---
 # Bug Report
 
+### Is the issue related to model conversion?
```

Comment:
Let's keep this is as a first section for now atleast. Right now we see a lot of issues related to model conversion and then we have to spend eng hours to redirect them to the right repo. IMO it would be best if we ask this as a question instead of note. Let's see if this reduces the number of converter issues we see in onnx. 
When the issue is not related to model conversion they can simply answer no and proceed to next section